Samples/Display recently visited products/Resources/RecentlyVisitedService.cs

```diff
@@ -0,0 +1,91 @@
+﻿using Litium.Products;
+using Litium.Runtime.DependencyInjection;
+using Litium.Web.Models.Products;
+using Newtonsoft.Json;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Web;
+
+namespace Litium.Accelerator.Services
+{
+    [Service(ServiceType = typeof(IRecentlyVisitedService), Lifetime = DependencyLifetime.Transient)]
```

Comment:
the service type is not needed to include in the implementation service, that will be inherited from interface.